pkg/patterns/declarative/reconciler.go

```diff
@@ -143,6 +150,38 @@ func (r *Reconciler) reconcileExists(ctx context.Context, name types.NamespacedN
 	if err != nil {
 		return reconcile.Result{}, err
 	}
+
+	var newItems []*manifest.Object
+	for _, obj := range objects.Items {
+
+		getOptions := metav1.GetOptions{}
+		gvk := obj.GroupVersionKind()
+		restMapper, err := apiutil.NewDiscoveryRESTMapper(r.config)
```

Comment:
As with the other PR, probably caching this (like you do dynamicClient) is a win